### ISSUE TITLE
Fusing Cydown with Cydownload

### ISSUE THREAD

**Issue by SquashingBugs** [borishonman/cydownload#7]
Cydown is a jailbreak tweak that enabled you to get paid cydia tweaks for free. If this could be included in Cydownload, that would be great!

**Comment by jinwk00** [comment]
Wanna get banned?

**Comment by IMS212** [comment]
No. This tweak is not for piracy, it is for people that need debs.

**Comment by ThePlayingGoblin** [comment]
Fuck off. I'm not paying money for tweaks.

**Comment by jinwk00** [comment]
Lol you spent hundreds of dollars on iDevice and you dont want to pay for
tweaks, congrats Mr. Pirate. We don't want you here.

**Comment by 459339321** [comment]
Where is the developer? Now I want to update this program. I don't know if the developer can satisfy the requirement because the download site changes the download rules.

**Comment by jinwk00** [comment]
The developer quit, piracy is illegal, so just buy the tweaks from the official source.

**Comment by 459339321** [comment]
I want to download the DEB of the designated site instead of the plug-in developer's pay package. Isn't the author Chinese?

**Comment by jinwk00** [comment]
Unless its a free Deb package, then I don't think there is anyway to save the deb files afaik

**Comment by 459339321** [comment]
他是一个免费分享的字体或者其他的包，被盗版源打包成DEB开启收费模式。不知道这样的算不算

**Comment by jinwk00** [comment]
I cannot understand that language smh

**Comment by 459339321** [comment]
I want to download the DEB of the designated site instead of the plug-in developer's pay package. Isn't the author Chinese?

**Comment by jinwk00** [comment]
I literally told you just now, haven't I?

**Comment by 459339321** [comment]
I'm sorry for the mistake I just made. He is a free-to-share font or other package, packaged by pirated sources into DEB to open charging mode. I don't know if that counts.

**Comment by jinwk00** [comment]
Always get the packages from the first party. Never trust the third party rehosts (which is essentially pirating).

**Comment by 459339321** [comment]
Apt. cydia. love, like this source, collects many free resources and opens the charging mode. The packages in the charging mode do not support ordinary users to download.

**Comment by debouncedgit** [comment]
> Always get the packages from the first party. Never trust the third party rehosts (which is essentially pirating).

https://github.com/jinwk00/cydownload/commit/1ccf4e5185367c412993cd958ab45828ce9cab14
not really fooling anyone by attempting to allow piracy via cydownload.

**Comment by UInt2048** [comment]
For historical reference, you can still find jinwk00's commit at: https://github.com/borishonman/cydownload/commit/1ccf4e5185367c412993cd958ab45828ce9cab14

**Comment by Lolguy1** [comment]
i just wanna express how much it baffles me that you guys are not supporting getting paid tweaks. the cydia store SHUTDOWN. this was made AFTER it shutdown so yall dont even have a good reason to reject the offer. like yes this tool that can very well get deb files, which can be good for archiving paid tweaks you can no longer get anymore too... lets make the people trying to archive stuff suffer. fuck you. yes, i can spend my money on an idevice but I LITERALLY CANT BUY THE TWEAK. the store is shutdown. devs have moved on. WHATS THE POINT OF NOT ALLOWING PAID TWEAKS OMFG. this is really enraging to me as i can no longer finally get the tweaks i want because its locked behind a paywall of a service THAT SHUTDOWN.
